source/js/table.bindings.js

```diff
@@ -0,0 +1,35 @@
+/* global ResponsiveTable, MobileTable */
+
+/**
+ * @file
+* Table component binding.
+ */
+(function (Drupal) {
+  'use strict';
+
+  Drupal.behaviors.gentBaseDefineTables = {
+    attach: function (context, settings) {
+
+      var tables = document.querySelectorAll('table');
+
+      for (var item = 0; item < tables.length; item++) {
+        var tableItem = tables[item];
+
+        // Create wrapper div and wrap it around each table
+        var tableWrapper = document.createElement('div');
+        tableWrapper.classList.add('responsive-table');
+
+        tableItem.parentNode.insertBefore(tableWrapper, tableItem);
+        tableWrapper.appendChild(tableItem);
+
+        // Adds accessibility support.
+        new ResponsiveTable(tableWrapper, {
+          scrollableText: '(scroll to see more)'
```

Comment:
```suggestion
          scrollableText: Drupal.t('(scroll to see more)')
```

And you should also add a translation to the `.po` file. 

_P.S. Didn't notice this in my last review, apologies 😇_

Reply:
Done :P